Modules/DataModel/Tests/PocketCastsDataModelTests/BookmarkDataManagerTests.swift

```diff
@@ -43,10 +43,10 @@ final class BookmarkDataManagerTests: XCTestCase {
     }
 
     func testAddingExistingBookmarkIsNotAdded() {
-        let firstUuid = addBookmark()
-        let secondUuid = addBookmark()
+        dataManager.add(episodeUuid: "episode-uuid", podcastUuid: "podcast-uuid", title: "Title", time: 1)
 
-        XCTAssertEqual(firstUuid, secondUuid)
+        let second = dataManager.add(episodeUuid: "episode-uuid", podcastUuid: "podcast-uuid", title: "Title", time: 1)
+        XCTAssertNil(second)
```

Comment:
I think it would be good to add a few additional checks that the current `episode-uuid` bookmark didn't have its title, time, etc. changed.

Reply:
Done here: 80e613ead30858da2dbe68e68f5edbb09356eb59